src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -288,28 +304,28 @@ public void changeListPanelView(TabName tabName) {
     }
 
     /**
-     * todo javadocs
+     * @return An Optional value containing the company list panel.
      */
     private Optional<ListPanel<? extends Item>> getCompanyTabView() {
         return Optional.of(new CompanyListPanel(companyItems));
     }
 
     /**
-     * todo javadocs
+     * @return An Optional value containing the application list panel.
      */
     private Optional<ListPanel<? extends Item>> setApplicationTabView() {
         return Optional.of(new ApplicationListPanel(applicationItems));
     }
 
     /**
-     * todo javadocs
+     * @return An Optional value containing the profile list panel.
      */
     private Optional<ListPanel<? extends Item>> setProfileTabView() {
         return Optional.of(new ProfileListPanel(profileItems));
     }
 
     /**
-     * todo javadocs
```

Comment:
Javadoc could perhaps start with Switches.... depending on

Reply:
resolved in the latest push.